.github/workflows/deploy-release.yml

```diff
@@ -14,20 +14,20 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.9
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip setuptools wheel babel
+          python -m pip install -U wheel hatch babel
```

Comment:
I'd recommend

```suggestion
          python -m pip install -U build babel
```

Reply:
I agree. Using `build` is the best way forward.

Reply:
But hold on, what about the `hatch publish` part?

Reply:
That's a good question. I haven't personally used `hatch publish`. I still use `twine` based solutions IIRC. I've only dipped my toes into `hatch. @ofek can probably answer better.

Reply:
Oh right, then you'd need Hatch still

Reply:
Anyway, why do I need `build`? Instead of trusting both `hatch` and `build` I could just be relying on `hatch`. In any case I trust `hatch` to continue doing the right thing going forward, more so than `build`

Reply:
I didn't notice publish step, you can revert (but still you don't need `wheel`)